package.json

```diff
@@ -24,7 +24,7 @@
     "@babel/runtime": "^7.2.0",
     "@cliqz/indexeddbshim": "^4.1.2",
     "appium": "1.9.0",
-    "browser-core": "https://s3.amazonaws.com/cdncliqz/update/edge/ghostery-ios/v3.36/1.36.0.ddb2925.tgz",
+    "browser-core": "https://s3.amazonaws.com/cdncliqz/update/edge/ghostery-ios/v3.36/1.36.0.c2d19d5.tgz",
```

Comment:
why `browser-core-lumen-ios` is not updated?

Reply:
good catch